doc/StatsConfig.md

```diff
@@ -34,11 +34,11 @@ Into file /etc/asterisk/logger.conf set the follow configs
 
 ```
 
-For /etc/asterisk/logger.conf set the follow configs 
+For /etc/asterisk/logger.conf & /etc/asterisk/extconfig.conf set the follow configs 
```

Comment:
There a problem here. The configuration  into /etc/asterisk/logger.conf is not necessary, it's wrong into this file.